Assets/Scripts/Core/Utils/VolumeControl.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.Audio;
+
+namespace OperationBlackwell.Core {
+    public class VolumeControl : MonoBehaviour {
```

Comment:
What does this class do?

Reply:
So this is used as a seperate component on the sliders? Should this not be directly integrated into the sliders directly?

Reply:
I can't add it directly in the sliders as the sliders are unity and I can't edit those? I'm not quite sure what you mean here.

Reply:
No never mind, what i had in thought does not work, as the updating of the mixer value happens differently then i thought.